app.py

```diff
@@ -19,6 +22,10 @@ async def root():
 async def get_name(name):
     return JSONResponse({"name": name}, status_code=200)
 
+@app.post("/name")
+async def post_name(request: NameRequest):
+    print("chandana")
```

Comment:
Choose add some logic (functionality) of code (ex: factorial of a given number).

Reply:
Sure